test/integration/repeated-slash-redirect/test/index.test.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-env jest */
+
+import cheerio from 'cheerio'
+import fs from 'fs-extra'
+import { join } from 'path'
+import {
+  launchApp,
+  killApp,
+  findPort,
+  nextBuild,
+  nextStart,
+  fetchViaHTTP,
+  renderViaHTTP,
+} from 'next-test-utils'
+import webdriver from 'next-webdriver'
+
+jest.setTimeout(1000 * 60 * 2)
+
+let appDir = join(__dirname, '..')
+const nextConfigPath = join(appDir, 'next.config.js')
+let nextConfigContent
+let appPort
+let app
+
+function getPathWithNormalizedQuery(url) {
+  const parsed = new URL(url, 'http://n')
+  if (parsed.search) {
+    parsed.search = `?${parsed.searchParams}`
+  }
+  return parsed.pathname + parsed.search
+}
+
+const runTests = (isDev = false) => {
+  const cases = [
+    ['/hello//world', '/hello/world'],
```

Comment:
Can we test `//hello/world`? 🙏

Reply:
I think I deliberately left it out to avoid issues with protocol-relative urls. But will add it back in and see what it does.

Reply:
@Timer Looks like I can make it work. However, it does mean that next.js will parse protocol-relative urls differently than browsers do. Not really a problem for me personally, but just so you're aware.